diff.go

```
@@ -13,7 +14,7 @@ type diffLine struct {
 	expected, actual any
 }
 
-func diffImpl(selectorPrefix string, expected, actual any) fun.Seq[diffLine] {
+func diffImpl(selectorPrefix string, expected, actual any) iter.Seq[diffLine] {
```

golangci-lint

Function name: diffImpl, Cyclomatic Complexity: 41, Halstead Volume: 7028.38, Maintainability Index: 14

(maintidx)